homeassistant/components/mystrom/light.py

```diff
@@ -89,12 +109,12 @@ def __init__(self, bulb, name, mac):
     @property
     def name(self):
         """Return the display name of this light."""
-        return self._name
+        return self._mac
 
     @property
     def unique_id(self):
         """Return a unique ID."""
-        return self._mac
+        return format_mac(self._mac)
```

Comment:
Is this not a breaking change?
Why format it now, when it wasn't formatted previously?